dubbo-registry/dubbo-registry-api/src/test/java/org/apache/dubbo/registry/support/FailbackRegistryTest.java

```diff
@@ -164,7 +164,7 @@ public void testDoRetry_nofify() throws Exception {
             public void notify(List<URL> urls) {
                 count.incrementAndGet();
                 //The exception is thrown for the first time to see if the back will be called again to incrementAndGet
-                if (count.get() == 1l) {

```

Comment:
why we need this? I believe either L or l can be used, both indicate a long primitive.

Reply:
In the Long type specification, it is recommended to use `L` instead of `l` because `l` and `1` are often confused.

Reply:
+1.